### DISCUSSION TITLE
CASE Tools

### DISCUSSION THREAD

**Discussion by zherar7ordoya** [community/community#116607] [Programming Help] [answered]
### Select Topic Area

Question

### Body

Hello people. CASE tools. At university we use Enterprise Architect. But there are others. I recently found out that the Eclipse universe has its own (some free and others paid). Rational Rose is very weighted. If anyone had the opportunity to use them, could you comment on your experience? Thank you.

**Comment by queenofcorgis** [comment, marked as answer]
Thanks for posting in the GitHub Community, @zherar7ordoya ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Reply by zherar7ordoya** [reply]
Thanks!
